infra/modules/gcp-host/output.tf

```diff
@@ -17,6 +17,10 @@ output "pseudonym_salt" {
   value       = module.psoxy.pseudonym_salt
 }
 
+output "service_accounts" {
```

Comment:
this is vague as to what these pertain to.

and especially when looking at it, they're just those for `api_connector`s, not bulk cases.

so I would name more clearly (`api_connector_gcp_execution_service_accounts`) and add a description; OR pull from `api_connector_instances`